ecs-microservice/grafana.tf

```diff
@@ -0,0 +1,29 @@
+###########################################################
+# Configure Grafana dashboard for microservice.
+#
+# Create a dashboard using the different values provided
+# as parameters to the mciroservice module to give a single
+# glass pane to watch the different components used by the
+# service.
+###########################################################
+
+data "aws_region" "current" {}
```

Comment:
Hva trengs denne til?

Reply:
den var ment til å sende inn current region til templaten, men er ikke i bruk for øyeblikket.